extensions/amp-analytics/amp-analytics.md

```diff
@@ -459,6 +459,30 @@ Use the timer trigger (`"on": "timer"`) to fire a request on a regular time inte
 }
 ```
 
+To configure a timer which times user events use:
+    - `startSpec` Specification for triggering when a timer starts. Use the value of `on` and `selector` to track specific events.
```

Comment:
Direct to the doc on how to specify a trigger event. Please also add documentation here that nested timer events are not supported.

Reply:
Done.